coral-hive/src/main/java/com/linkedin/coral/transformers/CoralRelToSqlNodeConverter.java

```diff
@@ -150,12 +160,12 @@ public Result visit(TableScan e) {
    */
   @Override
   public Result visit(Correlate e) {
-    final Result leftResult = visitChild(0, e.getLeft());
+    final Result leftResult = visitChild(0, e.getLeft()).resetAlias();
```

Comment:
generally curious - why did we need this change?

Reply:
While adding explicit alias, need to call this method to ensure the correct alias, actually it has been used in Coral-Trino: https://github.com/linkedin/coral/blob/master/coral-trino/src/main/java/com/linkedin/coral/trino/rel2trino/RelToTrinoConverter.java#L265